Jenkinsfile.promote

```diff
@@ -92,6 +92,53 @@ pipeline {
             }
         }
 
+        stage('Upload documentation and distribution') {
+            when {
+                expression { return isRelease() }
+            }
+            steps {
+                script {
+                    mavenCleanInstall('optaplanner', true, ['full'])
+                    dir('optaplanner') {
+                        withCredentials(bindings: [sshUserPrivateKey(credentialsId: 'optaplanner-filemgmt',
+                                             keyFileVariable: 'SSH_KEY_JBOSS_FILEMGMT')]) {
+                            // For testing, simulate connection via SSH:
+                            // sh "ssh -i $SSH_KEY_JBOSS_FILEMGMT -oKexAlgorithms=+diffie-hellman-group1-sha1 optaplanner@filemgmt.jboss.org"
+                            sh "./build/release/upload_distribution.sh ${getProjectVersion()} $SSH_KEY_JBOSS_FILEMGMT"
+                        }
+                    }
+                }
+            }
+        }
+
+        stage('Update OptaPlanner website') {
+            when {
+                expression { return isRelease() }
+            }
+            steps {
+                script {
+                    final String websiteRepository = 'optaplanner-website'
+                    dir(websiteRepository) {
+                        String prBranchName = createWebsitePrBranch(websiteRepository)
+
+                        // Update versions in links on the website.
+                        String nextMinorSnapshot = util.getNextVersion(getProjectVersion(), 'minor')
```

Comment:
why not using `getNextSnapshot` method ?

Reply:
'minor' vs. 'micro'. But let me do a small change.

Reply:
Created separate methods `getNextMicroSnapshotVersion` and `getNextMinorSnapshotVersion`.